### ISSUE TITLE
Encountered Crash Dump screen, followed by a Brick

### ISSUE THREAD

**Issue by PP-theSLAYER** [LumaTeam/Luma3DS#1112]
**System model:** Old 2DS

**SysNAND version (+emu/redNAND version if applicable):** Not sure. JP SysNAND

**Entrypoint (How/what you're using to boot Luma3DS):** Boot9Strap

**Luma3DS version:** Not Sure, can't check.

**Luma3DS configuration/options:** 
Enable game patching: (x)
Show NAND or user string in System Settings: (x)

--

**Explanation of the issue:**

Sadly, I'm unsure what version of Luma I was on, but I haven't updated to the newest Luma and didn't update to newest 2DS firmware. (I was probably on the previous FW version, since I was able to use Pokebank within the past month or so)

As a matter of fact, I was deleting titles using FBI before the brick, in preparation of the updates I was gonna do.
I know I didn't screw up title deletions, as I rebooted my 3DS a few times, before said BSOD.
If I'm not mistaken, I was exiting Rosalina (from inside Health and Safety).
I'm also certain I was on B9S.

Crash screen popped up, then 3DS was blue screen on reboot.
Attached to this message, is the latest Crash Dump I see in the Luma folder.

Text on the BSOD:
00F800FE
00000000 00000000
00000200 00000000

I'm kinda surprised that my 2DS managed to get bricked on B9S.
Ultimately, before certain users or people get triggered:
I'm not here to point fingers; bricks are a risk of using CFWs so I was mentally prepared for it.
Just want to prevent it from happening to others. I'll try to provide any other information, if needed.

TL:DR;
Crash message popped up as I was leaving Rosalina in H&S.
On reboot it became BSOD. The BSOD text changes per reboot.

**Steps to reproduce:**
N/A

**Dump file:**
[crash_dump_00000004.zip](https://github.com/AuroraWright/Luma3DS/files/2249561/crash_dump_00000004.zip)

**Comment by Multimegamander** [comment]
Do u have a NAND Dump, also, use fastboot3ds

**Comment by TuxSH** [comment]
What you use (b9s) is unrelated to the issue.

The crash dump is fs panicking.

00F800FE 00000000 00000000 00000200 00000000 is a "Error when having SD-card reader connected to NAND during boot"

This looks like a hardware issue to me, not a software one. Do or did you have a hardmod? If you can, could you check the motherboard?

In any case this is not a Luma3ds or software issue.

**Comment by PP-theSLAYER** [comment]
Thanks for the suggestion Multimegamander.

Based on the files on my 3DS's SD card, I have a NAND dump on it. (_NANDmin_sys.bin_)
(I have a habit of storing and labeling all my nand dumps in my external hard drive, but I'm overseas for the time being, so I don't have access to that external drive)

Additionally, I can't use fastboot3ds right now; NTRbootflasher claims that my Gateway Blue card is incompatible with ntrboot flashing, so I need to get a different NDS flash cart, in order to use ntrboothax to restore my NAND.


To add on, it looks like I'm not the only one affected by this issue.
I've read another one whether the use got bricked, the only difference is his/her's isn't BSOD,
potentially due the force shutting down of the system after a glitched screen appeared, when leaving Rosalina.

https://github.com/AuroraWright/Luma3DS/issues/1098

As such, I think it is worth the Devs' time to look at it.

**Comment by PP-theSLAYER** [comment]
Hey TuxSH, I have a hard mod (which I didn't want done but the guy did it. It was one of those guys that ghosted people for a long time on gbatemp, so I was just happy to get my console back), but I've never used it.
Plus, the cable-SD dongle never worked on my laptop, so I didn't bring it overseas (was afraid the customs would think it looks suspicious)


What specifically you want me to look for on the motherboard? It's a clear unit, so I easily look through the shell-housing

**Comment by TuxSH** [comment]
@PP-theSLAYER you won't be able to restore your nand because your nand is not correctly connected in the first place.

This is **not** a software issue, @Multimegamander suggestion is _not_ helpful.

**Comment by PP-theSLAYER** [comment]
You're saying my Nand was dislodged from the motherboard or something?

**Comment by TuxSH** [comment]
Oh hm thanks for the quick reply.

1) do the other person(s) who has/have this issue have had their hardmod done by the same person?

2) I think you should check these points: https://www.reboot.ms/forum/virtualdr/www.virtualdrive.cc/2DS_NAND_pinout.jpg , I'm asking more knowledgable ppl for that in the meanwhile.

> You're saying my Nand was dislodged from the motherboard or something?

Probably a contact failure or something like that. Currently the bootrom think you've connected the sd card bus to the nand pinouts (or something like that)

**Comment by PP-theSLAYER** [comment]
I've got no clue. My 2DS has been functional for at least a year since I've got it back, so never thought about it.

I probably got to take my 2DS apart, in other to see those points.

edit:
I don't have screw drivers that will allow me to open it up (once again, overseas so don't have the tools)
It's past midnight here, but I'm gonna check if any 24/7 marts sell skinny drivers that would work, so that I can take it apart.

At least from the outside, the wires doesn't seem to have shifted, but taking it apart will let us know for sure.
![img_9962](https://user-images.githubusercontent.com/23535344/43531334-e55f9044-95f2-11e8-801f-1dbf8e5137fd.JPG)

edit again: referring to https://github.com/AuroraWright/Luma3DS/issues/1098
@MrLucariox was your 3DS hard-modded?
How did you restore your Nand?

**Comment by PP-theSLAYER** [comment]
@TuxSH here are two shots of the cables on the motherboard.

If it's really the hardmod, would cutting off the wires solve the problem?

Also, there's a disconnected black wire. Not sure if it was disconnected during the opening, tho.
![img_9984](https://user-images.githubusercontent.com/23535344/43579967-2d454382-9697-11e8-8256-4fd6bac78bae.JPG)
![img_9985](https://user-images.githubusercontent.com/23535344/43579969-2d777fbe-9697-11e8-9be0-841850861a79.JPG)


edit:
The bright side is now, when I booted it with everything unscrewed, it actually turned on.
Down side is.. I don't know why. I haven't even removed the hardmod wires yet.

should I just snip off the wires and call it a day?

Also, now that it's not Luma, I say this can be closed :D

**Comment by pbanj** [comment]
ok so first let me say sorry it took me so long to reply here. my kid has been sick and yesterday was hectic. 

ok so you can go about solving this one of two ways. one you can try adding a little tape over the wires/points so they dont bridge against the screen. that black wire is ground for the hardmod, it being off means the hardmod wont work anymore. but it shouldnt cause an issue. from looking at your pics it looks as 2 wires got pinched and is most likely the cause of the issue as it looks like bare wires. so you can try and tape it like i said or snip all the wires of and out of the system. if you do that i would still add a little tape over the points just to be safe.

idk if you use discord but if you want to talk more about it you can reach me here @pbanj#9188 <that is my discord id

**Comment by PP-theSLAYER** [comment]
hey @pbanj , I'm sorry to hear that your kid is sick. Hope your kid gets well soon!

Thanks for the reply, I've already jumped the gun and cut off all the wires, since another one broke between the sessions of opening and closing the lid.

After I cut off the wires, the issue returned, and I realized that the back of the screen was partially conductive, so the small remnants of solder probably made it BSOD again. I taped the back of the screen, and everything peachy now! (tho I destroyed my rear camera ribbon by accident, but those are cheap on ebay, so not too concerned)

Anyhow, thanks for the advice, if I didn't already jump the gun, your advice would definitely been very useful in helping me figure out how to solve the calamity.

With that, this thoroughly proves that Luma isn't the issue here, as such I'm going to close this ><
